package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@nthparty/opaque",
-  "version": "0.3.0",
+  "version": "0.4.0-beta.1.0",
```

Comment:
Might be best to let the upstream maintainers decide what version this winds up being, likely in a separate PR as per [their usual pattern](https://github.com/nthparty/opaque/commits/main). There aren't any runtime changes here as far as I know, so I expect this would only be a semver-minor bump. Their deployment schedule might call for a beta tag anyway tho. I'd like not to make that decision for them unless I decide to deploy this fork to NPM separately.